demos/common/export_models/README.md

```diff
@@ -0,0 +1,102 @@
+# Exporting GEN AI models for deployments in OpenVINO Model Server
+
+This script automates exporting models from Hugging Faces hub or fine-tuned in PyTorch format to the models repository for deployment for the model serving.
+In one step it prepares a complete set of resources in the models repository for a supported GenAI use case.
+
+```bash
+pip install -q -r requirements.txt
```

Comment:
Our automatic tests would fail here - please add wget for test files or cloning of repository if this file is expected to be tested in automatic tests.